src/transformState.js

```diff
@@ -4,8 +4,29 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
Follow the checklist, please. Use switch/case 
<img width="793" alt="image" src="https://github.com/mate-academy/js_stateful-object/assets/99403198/aa2d7eab-044f-46bb-ab5f-3672fea6776b">

Reply:
You're right, sorry, I forgot to check it. I've just made the changes and seems to work too.